src/row.rs

```diff
@@ -0,0 +1,113 @@
+/// A macro for creating a [`Row`] using vec! syntax.
+///
+/// `row!` is similar to the [`vec!`] macro, but it returns a [`Row`] instead of a `Vec`.
+///
+/// # Examples
+///
+/// * Create a [`Row`] containing a vector of [`Cell`]s:
+///
+/// ```rust
+/// # use ratatui::prelude::*;
+/// use ratatui_macros::row;
+///
+/// let row = row!["hello", "world"];
+/// let row = row!["hello".red(), "world".red().bold()];
+/// ```
+///
+/// * Create a [`Row`] from a given [`Cell`] repeated some amount of times:
+///
+/// ```rust
+/// # use ratatui::prelude::*;
+/// use ratatui_macros::row;
```

Comment:
no need to repeat the import?

Reply:
I'm not sure I quite follow this comment.

Reply:
I meant that you only need to add the import once in the first example rather than repeating the same in each import.

Reply:
Is that the case? The doc tests error for me if I don't add it in every example.

Reply:
I mean that you don't need to show it - hide it after the first example block.

Reply:
Got it! I can make another PR for that across all the files.